cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certsecret.go

```diff
@@ -82,9 +84,15 @@ func createSecret(clusterApi kubernetes.Interface, namespace, secretName string,
 		},
 	}
 
-	_, err := clusterApi.CoreV1().Secrets(namespace).Create(secret)
+	err := wait.Poll(time.Second*5, time.Minute*3, func() (bool, error) {
```

Comment:
this change should be squashed into the previous commit please, it doesnt belong to this commit which say
build once and use binary

Reply:
Done

Reply:
aint you totally removed it ?